library/pod-security-policy/users/src.rego

```diff
@@ -0,0 +1,27 @@
+package k8spspallowedusers
+
+violation[{"msg": msg}] {
+  rule := input.parameters.runAsUser.rule
+  input_containers[input_container]
+  provided_user := input_container.securityContext.runAsUser
+  not accept_users(rule, provided_user)
```

Comment:
@ritazh @ctab what happens here when the securityContext is an empty object `{}` does this work ok?